compiler/src/dotty/tools/dotc/ast/Desugar.scala

```diff
@@ -820,7 +820,7 @@ object desugar {
       val moduleName = tdef.name.toTermName
       val localRef = Select(Ident(moduleName), tdef.name)
       localRef.pushAttachment(SuppressAccessCheck, ())
-      val aliasType = cpy.TypeDef(tdef)(rhs = completeForwarder(localRef))
+      val aliasType = cpy.TypeDef(tdef)(rhs = completeForwarder(localRef)).withSpan(Span(MaxOffset))
```

Comment:
Synthetic trees should always have a position where the start and the end are identical, that's how we distinguish them from user-written trees, but they should still point to the closest position where they were defined in the source code (otherwise, error messages involving them will point to the wrong line) so you should do `.withSpan(tdef.span.startPos)`

Reply:
Ok, this also works